test/test_variable_size_binary_layout.cpp

```diff
@@ -33,7 +33,7 @@ namespace sparrow
             m_data.buffers[1].resize(std::accumulate(
                 words,
                 words + nb_words,
-                0u,
+                size_t(0),
```

Comment:
note: c++23 adds the uz familly of litterals, which would have been appropriate here 🥲